ethereum/contracts/LightClientBridge.sol

```diff
@@ -291,10 +289,8 @@ contract LightClientBridge {
                 requiredNumOfSignatures
             );
 
-        /**
-         * @dev Encode and hash the commitment
-         */
```

Comment:
This created an error, probably a bug in the Solidity compiler: 

```
./ethereum/contracts/LightClientBridge.sol:295:17: ParserError: Only state variables or file-level variables can have a docstring.
        bytes32 commitmentHash =
                ^------------^

Compilation failed. See above.
Truffle v5.3.8 (core: 5.3.8)
Node v14.16.1
```

Reply:
weird

Reply:
Looks related to https://github.com/ethereum/solidity/issues/9139

Reply:
same here coming with v0.8.0 how to solve this ?